stackslib/src/net/api/mod.rs

```diff
@@ -127,6 +129,8 @@ impl StacksHttp {
         self.register_rpc_endpoint(poststackerdbchunk::RPCPostStackerDBChunkRequestHandler::new());
         self.register_rpc_endpoint(posttransaction::RPCPostTransactionRequestHandler::new());
         self.register_rpc_endpoint(getstackers::GetStackersRequestHandler::default());
+        self.register_rpc_endpoint(getsortition::GetSortitionHandler::new());
```

Comment:
Nit: can you keep these alphabetized by module?  Also applies to `getstackers` above.

Reply:
Updated in fdcaea8b91972ad721d9d9e0579ac89fb531ddc0